python/gtsam/tests/test_Sim2.py

```diff
@@ -27,10 +27,10 @@ def test_align_poses_along_straight_line(self) -> None:
         Scenario:
            3 object poses
            same scale (no gauge ambiguity)
-           world frame has poses rotated about x-axis (90 degree roll)
+           world frame has poses rotated about 180 degrees.
            world and egovehicle frame translated by 15 meters w.r.t. each other
         """
-        Rx90 = Rot2.fromDegrees(90)
+        R180 = Rot2.fromDegrees(180)
```

Comment:
@varunagrawal everything looks great. Was curious why you changed this value from 90 to 180, though. Both are valid, as I recall, for the purpose of the test.

Reply:
The translation values don't make sense otherwise. If we rotate by 90 degrees one of the two trajectories should change in the y axis.